x-pack/plugins/alerting/server/lib/snooze/is_snooze_active.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RRule, ByWeekday, Weekday, rrulestr } from 'rrule';
+import { RuleSnoozeSchedule } from '../../types';
+
+const MAX_TIMESTAMP = 8640000000000000;
+
+export function isSnoozeActive(snooze: RuleSnoozeSchedule) {
+  const { duration, rRule, id } = snooze;
+  if (duration === -1)
+    return {
+      id,
+      snoozeEndTime: new Date(MAX_TIMESTAMP),
+    };
```

Comment:
This whole file is just a copy-paste of code removed from `isRuleSnoozed`, except for this check for `duration === -1`

I'm temporarily having an indefinite snooze return an end time of `MAX_TIMESTAMP` just to avoid type errors and allow snoozes to be sorted by duration effectively. This timestamp is never parsed in the code because the UI and task runner still rely on the `muteAll` property. When we take care of https://github.com/elastic/kibana/issues/132266 we can look into whether passing `MAX_TIMESTAMP` for indefinite snoozes is the right thing to do, or if we want to handle this case differently.